packages/trace-viewer/src/ui/attachmentsTab.css

```diff
@@ -45,3 +45,9 @@
   max-width: 80%;
   box-shadow: 0 12px 28px 0 rgba(0,0,0,.2), 0 2px 4px 0 rgba(0,0,0,.1);
 }
+
+.attachment-item .attachment-item__text {
+  display: block;
+  width: 100%;
```

Comment:
Can you align the size of the text box with the content and make it not resizable?

Reply:
I was actually wondering whether I should use a `textbox` at all to display the attachment since it doesn't make sense to make it editable. I only did it for the easy resizing capabilities, but since you're suggesting to disable it, maybe I'll just switch to a styled `div`, WDYT?

Reply:
@yury-s any objections to changing from `textbox` to `div`? 🙂  otherwise I'll make the change

Reply:
Using `div` would be nice, but you'll need to preserve formatting and escape some characters, no objections otherwise.

Reply:
done [3520c7a](https://github.com/microsoft/playwright/pull/31215/commits/3520c7a86b7ac66e9fdead42021052c80bc7ee70)